spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/TaskLauncherTasklet.java

```diff
@@ -170,6 +176,10 @@ public RepeatStatus execute(StepContribution contribution,
 			this.executionId = taskOperations.launch(tmpTaskName,
 					this.properties, args);
 
+			Boolean ignoreExitMessage = isIgnoreExitMessage(args, this.properties);
```

Comment:
Should we try to use boot's binder to handle parsing a property as this looks quite low level to handle camelCase and kebab-case and args on top of that?